torchtitan/config_manager.py

```diff
@@ -451,6 +453,36 @@ class Float8:
     """
 
 
+@dataclass
+class MX:
+    # enable_fsdp_float8_all_gather: bool = False
```

Comment:
minor but why not just clear out all these commented out settings that are not valid for now, and add back in if/when it becomes usable?
would clean up the class quite a bit from a readability/usablity perspective, as there's only like 3 lines of actual usable settings rn.

Reply:
You are very right :)